docs/DeveloperGuide.adoc

```diff
@@ -236,6 +236,10 @@ Step 6. The user confirms the details of the person and `FormUiManager#setPerson
 
 Step 7. `FormUiManager` calls the `AddressBookParser#parseCommand()` to add the person and save it in the `Storage` interface.
 
+The following activity diagram summarizes what happens when you executes the `add t/**TYPE**` command:
+
```

Comment:
Line 229:
If you say the `Command` is a `Form` type, it is quite misleading because the word `type` is mostly used to refer to the type of person in our context. 
A suggested change: 
-Talk about the `CommandResult` having a boolean value `isForm`, which will be set to true. Then talk about the code that detects this boolean value and launches the `Form` mode.